package.json

```diff
@@ -2,28 +2,20 @@
   "name": "libnpmaccess",
   "version": "3.0.2",
   "description": "programmatic library for `npm access` commands",
-  "author": {
-    "name": "Kat Marchán",
-    "email": "kzm@zkat.tech",
-    "twitter": "maybekatz"
-  },
+  "author": "Kat Marchán <kzm@sykosomatic.org>",
   "license": "ISC",
   "scripts": {
     "prerelease": "npm t",
     "release": "standard-version -s",
     "postrelease": "npm publish && git push --follow-tags",
     "pretest": "standard",
-    "test": "tap -J --100 test/*.js",
-    "update-coc": "weallbehave -o . && git add CODE_OF_CONDUCT.md && git commit -m 'docs(coc): updated CODE_OF_CONDUCT.md'",
-    "update-contrib": "weallcontribute -o . && git add CONTRIBUTING.md && git commit -m 'docs(contributing): updated CONTRIBUTING.md'"
+    "test": "tap -J --100 test/*.js"
```

Comment:
Minor changes, based on isaacs feedback on other PR's:
Let's this be just `"test": "tap"`, add a section like:
```
{
  "tap": {
    "check-coverage": true
  }
}
```
and rename test/util to test/fixtures so that tap will ignore it.
(ex. https://github.com/npm/libnpmsearch/blob/latest/package.json)